Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-current-contest.tsx

```diff
@@ -33,6 +33,7 @@ interface ICurrentContestContext {
         submitContestPasswordErrorMessage: string | null;
         isPasswordValid: boolean | null;
         remainingTimeInMilliseconds: number;
+        validation: IStartParticipationValidationType;
```

Comment:
`validation` is incorrect naming. Think of a better one

Reply:
Renamed to validationError.